examples/mfa/example.go

```diff
@@ -41,62 +49,66 @@ func main() {
 		RedirectURI:  redirectURI,
 	}
 
-	opk := &client.OpkClient{
-		Op: provider,
+	cosignerProvider := client.CosignerProvider{
```

Comment:
Am I missing something, is `client.CosignerProvider` getting instantiated somewhere above? Otherwise I'm pretty sure this will throw an error

Reply:
I might be misunderstanding you. `client` is the name of the package, so `client.CosignerProvider{...}` is creating a new CosignerProvider struct.

Reply:
oh right, yeah I misunderstood thanks!